### DISCUSSION TITLE
In Smart ChatGPT the copy button doesn't work

### DISCUSSION THREAD

**Discussion by ottantaventi** [brianpetro/smart-connect#60] [Help & Support]
When using the Smart ChatGPT and clicking on the copy button of ChatGPT i get an error message and it doesn't copy the text of the response

![image](https://github.com/user-attachments/assets/21b6eadf-dfa3-44a0-b3e5-d6f3713f4306)

**Comment by brianpetro** [comment]
Unfortunately, I've tried to fix this multiple times, and just made another attempt, to no avail.

Fortunately, highlighting the text and manually copy&pasting still works.

🌴

**Reply by ottantaventi** [reply]
Thank you for the feedback
